state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -626,14 +639,31 @@ pub mod pallet {
 		DepositBoosted {
 			deposit_address: TargetChainAccount<T, I>,
 			asset: TargetChainAsset<T, I>,
-			amount: TargetChainAmount<T, I>,
+			amounts: BTreeMap<BoostPoolTier, TargetChainAmount<T, I>>,
 			deposit_details: <T::TargetChain as Chain>::DepositDetails,
+			channel_id: ChannelId,
 			// Ingress fee in the deposit asset. i.e. *NOT* the gas asset, if the deposit asset is
 			// a non-gas asset.
 			ingress_fee: TargetChainAmount<T, I>,
 			boost_fee: TargetChainAmount<T, I>,
 			action: DepositAction<T::AccountId>,
 		},
+		BoostFundsAdded {
+			account_id: T::AccountId,
+			boost_pool: BoostPoolId<T::TargetChain>,
+			amount: TargetChainAmount<T, I>,
+		},
+		AccountStoppedBoosting {
+			account_id: T::AccountId,
+			boost_pool: BoostPoolId<T::TargetChain>,
+			account_available_amount: TargetChainAmount<T, I>,
+			account_in_use_amounts: BTreeMap<BoostId, TargetChainAmount<T, I>>,
```

Comment:
A comment describing why we have two sets of amounts here would be useful for future readers